### ISSUE TITLE
Net imgui

### ISSUE THREAD

**Issue by SlackingVeteran** [IDI-Systems/UnrealImGui#16]
Is there any plan to integrate net-imgui into master and allow plugin users to decide if they want to enable net-imgui functionality?

**Comment by jonpas** [comment]
Yes, there is a pull request #1 open, however time in day is limited. :)
